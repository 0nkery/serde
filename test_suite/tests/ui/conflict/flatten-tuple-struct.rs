#[macro_use]
extern crate serde_derive;

use std::collections::HashMap;

#[derive(Serialize)]
struct Foo(u32, #[serde(flatten)] HashMap<String, String>);

fn main() {}
