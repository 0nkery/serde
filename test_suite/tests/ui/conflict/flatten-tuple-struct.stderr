error: #[serde(flatten)] cannot be used on tuple structs
 --> $DIR/flatten-tuple-struct.rs:7:17
  |
7 | struct Foo(u32, #[serde(flatten)] HashMap<String, String>);
  |                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error: aborting due to previous error

